src/admin/resolvers/queries/RejectedItem.auth.integration.ts

```diff
@@ -0,0 +1,145 @@
+import { db } from '../../../test/admin-server';
+import {
+  clearDb,
+  createRejectedCuratedCorpusItemHelper,
+  getServerWithMockedHeaders,
+} from '../../../test/helpers';
+import { ACCESS_DENIED_ERROR, MozillaAccessGroup } from '../../../shared/types';
+import { GET_REJECTED_ITEMS } from './sample-queries.gql';
+import { expect } from 'chai';
+
+describe('queries: RejectedCuratedCorpusItem (authentication)', () => {
+  // A few sample Rejected Curated Corpus items - we don't need a lot of these
+  // for auth checks
+  const rejectedCuratedCorpusItems = [
+    {
+      title: '10 Unforgivable Sins Of PHP',
+    },
+    {
+      title: 'Take The Stress Out Of PHP',
+    },
+    {
+      title: 'The Untold Secret To Mastering PHP In Just 3 Days',
+    },
+    {
+      title: 'You Can Thank Us Later - 3 Reasons To Stop Thinking About PHP',
+    },
+  ];
```

Comment:
Added only a handful of rejected corpus items for this suite of tests as we don't need many to make sure users get to see all of the Rejected Corpus if they are entitled to do so, and we don't need to run queries narrowed down by language or any other fields or queries limited by pagination - all of this is tested thoroughly in existing integration tests.

Also, following an emerging pattern from https://github.com/Pocket/curated-corpus-api/pull/567 by placing auth-related integration tests into its own file.